templates/tab/ts/default/.env.teamsfx.local

```diff
@@ -0,0 +1,7 @@
+# Following variables will be overwritten by TeamsFx
```

Comment:
Why do we need this file in source control?

Reply:
Let users understand where the environment variable values come from when running the code locally. This will be more straightforward than asking users to view the package.json file. And many samples (e.g. the botframework samples) provides an .env to help user get started easily.

Reply:
This file will be re-generated by debug component when user try local debug. The updated .env.teamsfx.local will be:
```
# Following variables are generated by TeamsFx
BROWSER=none
HTTPS=true
SSL_CRT_FILE=xxxxxx/.fx/certificate/localhost.crt
SSL_KEY_FILE=xxxxxx/.fx/certificate/localhost.key
REACT_APP_START_LOGIN_PAGE_URL=https://localhost:53000/auth-start.html
REACT_APP_CLIENT_ID=xxxxxxxxxxx
REACT_APP_FUNC_ENDPOINT=
REACT_APP_FUNC_NAME=
PORT=53000

# Following variables can be customized or you can add your owns
# FOO=BAR
```
Is it expected?

Reply:
Thanks. I think we need to use your version (and remove the values) to ensure we have all the keys. This also reminds me that @swatDong is updating the wording in this file. @formulahendry can you help followup with Qianhao on the final content?

Reply:
FYI. The wordings can be found in both source code and [doc](https://github.com/OfficeDev/TeamsFx/blob/dev/docs/fx-core/localdebug-help.md#what-to-do-if-i-want-to-add-my-own-environment-variables-for-local-debug).

Reply:
Thanks all for the valuable comments~